guides/common/modules/con_resolving-package-dependencies.adoc

```diff
@@ -1,56 +1,67 @@
+Dependency solving can lead to the following problems:
 
-If you create a content filter for security purposes but choose to resolve dependencies, {Project} can add packages that you might consider insecure.
+Dependency solving can significantly delay Content View publication::
+{Project} examines every repository in a Content View for dependencies.
+Therefore, publish time increases with more repositories.
++
+To mitigate this problem: Use multiple Content Views with fewer repositories.
+Combine them using composite Content Views.
 
-.Filtering and Dependency Solving Recommendations
-Be mindful when combining exclusion filters with dependency solving.
+{Project} does not apply content filters on dependent packages::
+{Project} prioritizes resolving package dependencies over the rules in your filter.
++
+Example: If you create a filter for security purposes but enable dependency solving, {Project} can add packages that you might consider insecure.
++
+To mitigate this problem: Carefully test filtering rules to determine the required dependencies.
+If dependency solving includes unwanted packages, manually identify the core basic dependencies that the extra packages and errata need.
 ifeval::["{client-os-family}" == "Red Hat"]
-Consider this example: You recreate {RHEL} 8.3 via Content View filters and include select errata from a later {RHEL} 8 y-release.
-To achieve this, you exclude all errata after the {RHEL} 8.3 release date minus some select few that are needed, and enable dependency solving.
-Dependency solving may pull in more packages than expected, which would cause the client to stray further from being a {RHEL} 8.3 machine.
-endif::[]
-If dependency solving pulls in too many unwanted packages, consider identifying by hand the core basic dependencies that the extra packages and errata need.
-ifeval::["{client-os-family}" == "Red Hat"]
-If, in the above case, the user does not need the extra errata and packages, then Content View filtering is not needed at all.
-You can enable and use the {RHEL} 8.3 repository on the *Content* > *Red Hat Repositories* page in the {ProjectWebUI}.
+See also xref:Combining_exclusion_filters_with_dependency_solving[].
 endif::[]
 
 ifeval::["{client-os-family}" == "Red Hat"]
-In some cases, excluding packages may make dependency solving impossible for DNF.
-For example, if you make a {RHEL} 8.3 repository with a few excluded packages, `dnf update` could fail.
-Instead of turning on dependency solving to resolve the issue, investigate the error from DNF and adjust the filters to stop excluding the missing dependency.
-If dependency solving is enabled, the repository may stray from {RHEL} 8.3 as discussed above.
-endif::[]
+[id="Combining_exclusion_filters_with_dependency_solving"]
+.Combining exclusion filters with dependency solving
```

Comment:
This should probably go into separate module.

Reply:
I know what you mean, that was my first instinct too. I chose to keep it all within one concept module for these reasons:

* I don't think the two modules would ever be reused separately. They should always appear together -- if you're reading about dependency solving overview (the first part of the module), you will need to follow up with reading about filtering and dependency solving.
* The first part of the module links to the second part about the filters; so they really are very closely related.
* The procedure module below (Enabling Dependency Solving) needs to point to the concept explanation to warn users about the pitfalls of the feature. I'd rather point to one concept module rather than two.

Reply:
Well I won't object to keeping it in the same module but I would like to know what other writers think about this. @maximiliankolb @Lennonka

Reply:
I agree with Aneta's view when those modules are not going to be reused and are strongly bonded to the primary procedure.

Reply:
@Lennonka commented elsewhere, advised to use one more example block.

So I think it's safe to resolve this thread.